.github/workflows/publish_releasenotes_for_twitter.yml

```diff
@@ -18,4 +18,10 @@ jobs:
         uses: actions/checkout@v2
       - name: Run Shell Script
         run: |
-            ./.ci/tweet-releasenotes.sh ${{ github.event.inputs.version }}
+          ./.ci/tweet-releasenotes.sh ${{ github.event.inputs.version }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
What happens if you remove this line ?
We do not need redefine GitHub token.

Reply:
Without redefining the GitHub Token and removing this line from workflow: It [FAILS](https://github.com/Rahulkhinchi03/checkstyle/runs/8084952250?check_suite_focus=true)

Reply:
Also found that we do not need to create a secret for PERSONAL_ACESS_TOKEN.

Using `${{ secrets.PERSONAL_ACCESS_TOKEN }}` will automatically creates the token and uses it so we do not need to create a secret for it.

Reply:
The code is updated only.

Reply:
ok, update code with usage of `GITHUB_READONLY_TOKEN` env variable. We will put there personal token for now.

Reply:
update tweet-releasenotes.sh to use GITHUB_READONLY_TOKEN to make it clear that it is not general GITHUB_TOKEN

Reply:
Changes Done